JAX to HLO/jax_to_hlo.py

```diff
@@ -0,0 +1,199 @@
+# Copyright 2019 Google LLC
```

Comment:
It's not 2019 any more ;)

We should probably also pay attention to Section 4b of the license under which you're copying this.

Reply:
I see you've already added a note about where this code comes from, in a subsequent commit.

Reply:
I just made a simpler example without needing this extra file, so resolving the comments for this file.